synapse/rest/client/register.py

```diff
@@ -929,6 +929,10 @@ def _calculate_registration_flows(
         # always let users provide both MSISDN & email
         flows.append([LoginType.MSISDN, LoginType.EMAIL_IDENTITY])
 
+    # Add a flow that doesn't require any 3pids, if the config requests it.
+    if config.registration.enable_registration_token_3pid_bypasss:
+        flows.append([LoginType.REGISTRATION_TOKEN])
```

Comment:
Should this have been:
```
flows.append([LoginType.REGISTRATION_TOKEN, LoginType.DUMMY])
```

...for similar reasons as stated [here](https://github.com/matrix-org/synapse/blob/78b99de7c206b106340e12cdee0af9aa246bd5ad/synapse/rest/client/register.py#L914-L917)?